CMakeLists.txt

```diff
@@ -531,6 +531,13 @@ endif(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
         MESSAGE(FATAL_ERROR "=================================\n"
                             "Python not found, install Python!\n"
                             "=================================\n")
+    ELSE(NOT PYTHONLIBS_FOUND)
+        # prevent python3 lower than 3.3 (not enough utf8<->unicode tools)
+        IF(PYTHON_VERSION_MAJOR GREATER 3)
+            IF(PYTHON_VERSION_MINOR LESS 5)
+                MESSAGE(FATAL_ERROR "To build FreeCAD with Python3, you need at least version 3.5\n")
+            ENDIF(PYTHON_VERSION_MINOR LESS 5)
+        ENDIF(PYTHON_VERSION_MAJOR GREATER 3)
```

Comment:
Umm, if Python 2.7.x is supported, shouldn't this and the `LESS 5` line above it be for 2.7 and 3.5+ ? And the comment of the `ELSE(NOT` line this code block is in doesn't feel right either.